app/api/models.py

```diff
@@ -1,8 +1,8 @@
 """Data models."""
-
 from typing import Optional, Union
 
-from pydantic import BaseModel
+from fastapi import Query
+from pydantic import BaseModel, Field
```

Comment:
isort? mind opening an issue to add this to the CI?

Reply:
isort is already included in the precommit hooks for this repo! 🙂

https://github.com/neurobagel/federation-api/blob/f85c44b22266fa7ebece037bd624b880b4e85355/.pre-commit-config.yaml#L25-L29

I think this is the expected import order following
1. standard library imports
2. related third party imports
3. local application/library specific imports